.do/Makefile

```diff
@@ -37,7 +38,8 @@ define call_helm_install
 	--set crow.config.passwordAuth.password=$${CROW_PASSWORD} \
 	"$${extra_args[@]}"
 
-  $(foreach DEPLOYMENT, $(DEPLOYMENTS),$(call rotate_pod, $(DEPLOYMENT)))
+  $(foreach DEPLOYMENT, $(DEPLOYMENTS),$(call rotate_pod, $(DEPLOYMENT), $(NAMESPACE)))
+  $(foreach DEPLOYMENT, $(MONITORING_DEPLOYMENTS),$(call rotate_pod, $(MONITORING_DEPLOYMENTS), "monitoring"))
```

Comment:
We do this because the rabbitmq dashboard gets created after grafana has been deployed to the cluster. 
Therefore, we need to restart grafana so it picks up the configmaps containing additional dashboards.